contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,172 @@
+
+1. Consume the raw events from the same Kafka topic as our ingestion service. 
+2. Stream them to you ASAP using server-sent events. 
+
+The lack of person data is a fair trade-off, because it doesn’t get more live than this. User activity appears within milliseconds of happening on the other side of the world.
+
+Speaking of data being processed live, perhaps at some point you’ve wondered if PostHog actually scales, or just talks the talk. To dispel any doubts, we developed one extra feature: an anonymized stream of all events being captured globally, with only the geolocation included. Hundreds of thousands of events per minute, and they look great on a 3D globe on PostHog’s website, where each event is an arc from the user’s location to that of the relevant data center. Global scale, visualized for your pleasure.
+
+
+### SupportHog
+
+<BuiltBy people={["Dylan Martin", "Tiina Turban", "Marcus Hof", "Neil Kakkar"]} />
+
+Our support flow currently uses Zendesk and that goes through email. This causes three problems: emails sometimes bounce, there are long delays in checking emails, and it’s really clunky to add other team members via Cc. Simply put, it’s not the most optimal flow. 
+
+Imagine if instead users could view and respond to their open support tickets without ever having to leave PostHog – well, that's exactly what we built.
+
+But that's not all, if you're using Zendesk then you could, in the future, add this view to your customer-facing website with just a few clicks. Like the sound of this project? [Check out our roadmap](https://posthog.com/roadmap) and vote for it.
+
+### The ~pyramid~ referral scheme
+
+<BuiltBy people={["Ben White", "Raquel Smith", "Joe Martin"]} />
+
+Everyone loves a pyramid scheme, right?! Wait, no, we mean a _referral program._ Ben, Raquel, and Joe worked together to build a referral product right into PostHog. This means we can offer sweet merch, platform credits, good vibes, and other things to loyal PostHog users who lure in their friends and family. 
+
+What’s more, as the referral program product is built right into PostHog, you can build your own for your customers. The system is hooked up to Zapier to automate the process for redeeming codes making the whole thing a doddle. It’s not shipped yet, but we’re close.
+
+### A/B TestHog
+
+<BuiltBy people={["Ian Vanagas", "Juraj Majerik", "Lior Neu-ner"]} />
+
+
+![A/B TestHog](https://res.cloudinary.com/dmukukwp6/image/upload/abtesthog_e3f93c1afc.png)
+
+Want to know how to improve your website but don’t know where to start? You need A/B TestHog. Enter a website URL, click ‘Analyze’ and an ingenious generative AI system will give you a bunch of recommendations for what A/B tests you might run to take your website to the next level. These are all expertly authored by an AI, and include the goal metrics, secondary metrics, and guardrail metrics and detailed instructions of what to change for your test.
```

Comment:
Can we get this live @ivanagas? Would be cool if people could try it, even if it's "proof of concept".